network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,137 @@
+---
+meta:
+  title: Configuring your Load Balancer
+  description: Learn how to choose different settings and configurations for your Load Balancer depending on your needs
+content:
+  h1: Configuring your Load Balancer
+  paragraph: Learn how to choose different settings and configurations for your Load Balancer depending on your needs
+tags: load-balancer
+categories: 
+  - load-balancer
+  - network
+---
+
+## Load Balancer Overview
+
+Load Balancers allow you to distribute traffic and workload across multiple backend servers. They make it easy to scale your applications while ensuring continuous availability even in the event of heavy traffic. Each Load Balancer that you create is a highly available and fully managed Instance, consisting of one or more frontends and one or more backends. You can configure your Load Balancer according to your needs.
+
+<Lightbox src="scaleway-load-balancer.webp" alt="" />
+
+In this document, we explain some of the different configurations available when setting up your Load Balancer, and give examples of how to configure its frontends and backends for different use cases.
+
+## Configuring a frontend
+
+The frontend listens on the port you specify, and forwards received requests to the backend(s)
+
+When creating a frontend, aside from the name the only configurable setting is that of the **port** it listens on.
+
+### Choosing a port
+
+For HTTP, a frontend will typically listen on port 80, while for HTTPS it will typically listen on port 443. 
+
+After creating your Load Balancer, you can create multiple frontends to listen on different ports, and connect them to different backends as required. You can also set up custom [ACL rules](/network/load-balancer/how-to/create-manage-acls/) for each frontend.
+
+## Configuring a backend 
+
+The configuration that you choose for your Load Balancer's backend determines the manner in which it transfers requests to its associated backend servers.
+
+A number of configuration settings are available.
+
+<Message type="note">
+
+  Here, the term "backend" refers to a set of configurations that determine how the Load Balancer forwards traffic. You can create one or more backends for your Load Balancer. The term "backend server" designates the final-destination backend servers which the Load Balancer forwards traffic on to. Each backend must specify one or more backend servers (via their IP addresses) to forward to.
```

Comment:
I think the backend definition is still a bit confusing. I think the primary definition is that the backend is a group of backend servers (forwarding targets), and it also carries configuration applied to all backend servers. All backend servers in the backend should be able to serve the same requests in the same way (the backend can be seen as a service we are forwarding traffic to).

Reply:
is the change I've made OK for you?

Reply:
Good to me, thanks